src/javascript/esutil.ts

```diff
@@ -171,26 +171,29 @@ function getLeadingComments(node: estree.Node): string[]|undefined {
 }
 
 export function getPropertyValue(
-    node: estree.ObjectExpression, name: string): estree.Node|undefined {
+    node: babel.ObjectExpression, name: string): babel.Node|undefined {
   const properties = node.properties;
   for (const property of properties) {
-    if (objectKeyToString(property.key) === name) {
+    // TODO(usergenic): Can't get property key and value from a SpreadProperty.
```

Comment:
let's start filing issues for TODOs

Reply:
We can do a little of this with scope analysis. If we can resolve the expression to a literal we can get the keys/values from the literal.